docs/UserGuide.md

```diff
@@ -157,13 +163,8 @@ Examples:
 * `findPatient n/ John` returns patients with names `john` and `John Doe`
 * `findPatient n/ alex david` returns patients with names `Alex Yeoh`, `David Li`<br>
 * `fp n/ alex david` returns patients with names `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
 
-### Listing all patients : `list`
-
-Shows a list of all patients in CLInic.
-
-Format: `list`
+  ![result for 'find alex david'](images/findAlexDavidResult.png)
```

Comment:
LGTM! Just wondering, should we add a caption about how captilisation of DaviD as in the picture will not hinder using find?

Reply:
Hmmm maybe since its a user guide it may not be that important? seems like something that should be in developer guide (design considerations) instead, but i think it would also be helpful if it was in the user guide - just a bit worried that it might be too complicated to read if we include it for other commands as well (like specify which ones need capitalisation which ones dont need)